### PR TITLE
XRT-2063: Add multiple service examples

### PR DESCRIPTION
**Pull request by Tylerg2471**

### CODE REVIEW

File: DeviceServices/multiple-services/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+## Overview
+
+This page shows you how to run XRT with multiple device services. The following device services are included in this example:
+* BACnet/IP
+* OPC-UA
+* Modbus-TCP
+
+
+For more information about the Device Services included please review the [BACnet Device Service](https://docs.iotechsys.com/edge-xrt20/device-service-components/bacnet-device-service-component.html), [Modbus Device Service](hhttps://docs.iotechsys.com/edge-xrt20/device-service-components/modbus-device-service-component.html) and [OPC-UA Device Service](https://docs.iotechsys.com/edge-xrt20/device-service-components/opc-ua-device-service-component.html) documentation.
```
Comment (BinduRao2018) [inline DeviceServices/multiple-services/README.md:11]: links are pointing to 2.0

File: DeviceServices/multiple-services/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+Set the environment variable `DIR_PATH` to the `DeviceServices` path in the xrt-examples directory, for example:
+
+``` shell
+    export DIR_PATH=/xrt-examples/DeviceServices
+```
+
+### Setup Simulators
+
+For each service included, setup the relevant simulator and envvironment variables by following the instructions in the individual device service guides.
```
Comment (BinduRao2018) [inline DeviceServices/multiple-services/README.md:25]: typo : environment

**Review by BinduRao2018 — CHANGES_REQUESTED** [review]
Comments in-line.
A link to the multi-services example should be added to Examples section in DeviceServices/Readme.md

File: DeviceServices/multiple-services/README.md (hunk 2 above)
Comment (Tylerg2471) [reply DeviceServices/multiple-services/README.md]: Fixed

File: DeviceServices/multiple-services/README.md (hunk 1 above)
Comment (Tylerg2471) [reply DeviceServices/multiple-services/README.md]: Fixed

**Comment by BinduRao2018** [comment]
@Tylerg2471, you have missed this:
'A link to the multi-services example should be added to Examples section in DeviceServices/Readme.md'

**Comment by Tylerg2471** [comment]
> @Tylerg2471, you have missed this: 'A link to the multi-services example should be added to Examples section in DeviceServices/Readme.md'

Fixed now

**Review by BinduRao2018 — APPROVED** [review]

**Pull request merged**